src/clj/fluree/db/flake.cljc

```diff
@@ -575,8 +534,9 @@
   reference: https://www.javamex.com/tutorials/memory/string_memory_usage.shtml"
   [^Flake f]
   (let [o      (o f)
-        dt     (int (dt f))
-        o-size (util/case+ dt
+        dt     (dt f)
+        o-size (#?@(:clj (util/case+)
+                    :cljs (condp =)) dt
                  const/$xsd:string (* 2 (count o))
                  const/$xsd:anyURI 8
```

Comment:
Are sids uniform in length anymore? It doesn't seem like it to me, so I'm not sure that `size-flake` is calculating correctly anymore.

Reply:
They're not. I missed this. Thanks for pointing it out.